### PR TITLE
Simon HBV (supersedes #102)

### PR DESCRIPTION
**Pull request by simonleandergrimm**
This pull request supersedes #102. It includes:
 - Asher's original addition total (acute + chronic) reported Hep B cases from 2021 in Ohio, now split into acute and chronic cases.
 - Added a CDC underreporting factor to scale up reported acute cases from Ohio.
 - Added CDC-provided reported chronic Hep B cases.

### CODE REVIEW

File: pathogens/hbv.py [hunk 1]
```diff
@@ -7,15 +7,22 @@
...
 # - Select time period of death": 2022, and
-# - Slect underlying cause of death: B16 (Acute hepatitis B).
+# - Select underlying cause of death: B16 (Acute hepatitis B).
 # All other fields can be left as default.
+# 3. I currently lean against arriving at an underreporting factor for chronic
+# infections, as this would eat up too much time and looks too hacky on first
+# glance. If I don't calculate such a factor, I would  drop reported chronic
+# infections, as the estimated prevalence  numbers provided by
+# `estimated_chronic_prevalence_us_2020` provide a cleaner picture of overall
+# chronic Hep B disease burden.
```
Comment (simonleandergrimm) [inline pathogens/hbv.py:25]: @jeffkaufman Let me know if you agree.

File: pathogens/hbv.py [hunk 2]
```diff
@@ -58,17 +91,41 @@
...
 
 
 def estimate_incidences() -> list[IncidenceRate]:
-    return [cdc_estimated_acute_2019.to_rate(us_population_2019)]
+    return [
+        cdc_estimated_acute_2019.to_rate(us_population_2019),
+        cdc_reported_chronic_2019,
+        ohio_reported_acute_incidence_2021
+        * cdc_acute_underreporting_factor_2020,
+        ohio_reported_chronic_incidence_2021,
```
Comment (jeffkaufman) [inline pathogens/hbv.py:130]: This doesn't seem right to me -- why aren't we summing the acute and chronic cases?

**Review by jeffkaufman — CHANGES_REQUESTED** [review]

File: pathogens/hbv.py (hunk 2 above)
Comment (simonleandergrimm) [reply pathogens/hbv.py]: Discussed in person

File: pathogens/hbv.py [hunk 3]
```diff
@@ -58,17 +77,29 @@
...
+    return [
+        estimated_chronic_prevalence_us_2020.to_rate(us_population(year=2020))
+    ]
 
 
 def estimate_incidences() -> list[IncidenceRate]:
-    return [cdc_estimated_acute_2019.to_rate(us_population_2019)]
+    return [
+        cdc_estimated_acute_2019.to_rate(us_population_2019),
+        cdc_reported_chronic_2019,
```
Comment (jeffkaufman) [inline pathogens/hbv.py:102]: Where is this defined?  Does this change pass tests?

**Review by jeffkaufman — CHANGES_REQUESTED** [review]

**Review by jeffkaufman — APPROVED** [review]

**Pull request merged**